main.go

```diff
@@ -436,6 +439,32 @@ func decodeGetMapForts(ctx context.Context, sDec []byte) string {
 	return "No forts updated"
 }
 
+func decodeGetRaidLobbyCounter(ctx context.Context, sDec []byte) string {
+	decodedRaidLobbyCounter := &pogo.GetRaidLobbyCounterOutProto{}
+	if err := proto.Unmarshal(sDec, decodedRaidLobbyCounter); err != nil {
+		log.Errorf("Failed to parse %s", err)
+		return fmt.Sprintf("Failed to parse %s", err)
+	}
+
+	if decodedRaidLobbyCounter.Result != pogo.GetRaidLobbyCounterOutProto_SUCCESS {
+		return fmt.Sprintf(`GetRaidLobbyCounterOutProto: Ignored non-success value %d:%s`,
+			decodedRaidLobbyCounter.Result,
+			pogo.GetRaidLobbyCounterOutProto_Result_name[int32(decodedRaidLobbyCounter.Result)])
```

Comment:
Will produce the same result while being simpler to read. 
```suggestion
			decodedRaidLobbyCounter.Result.String())
```

Reply:
Nice, thanks!